### PR TITLE
Start using more consistent logging.

### PR DESCRIPTION
**Pull request by charlie-ht**
I didn't want to comb through all the logging in one sitting, so I propose getting a review on the direction. The result of the old logging can be removed piece by piece.

Part of this includes removing code that was printing raw bytes to C
streaming I/O functions. Bytes mean different things there (NUL means
end of string for example), so this method of printing is not going to
work properly.

### CODE REVIEW

File: src/browser/wpe/opencdm/open_cdm.cpp [hunk 1]
```diff
@@ -161,38 +157,32 @@ int OpenCdm::Load(std::string& responseMsg) {
   return ret;
 }
 
-int OpenCdm::Update(unsigned char* pbResponse, int cbResponse, std::string& responseMsg) {
-
-  CDM_DLOG() << "Update >> invoked from ocdm :: estate = " << m_eState << "\n";
+int OpenCdm::Update(unsigned char* pbResponse, int cbResponse, std::string& responseMsg)
+{
```
Comment (calvaris) [inline src/browser/wpe/opencdm/open_cdm.cpp:161]: This code seems to have incoherent style.

File: src/browser/wpe/opencdm/open_cdm.cpp [hunk 2]
```diff
@@ -250,86 +240,82 @@ int OpenCdm::ReleaseMem() {
...
-
-  CDM_DLOG() << "OpenCdm::ReadyCallback";
+  CDM_LOG_LINE("call over");
 }
 
-void OpenCdm::ErrorCallback(OpenCdmPlatformSessionId platform_session_id,
-    uint32_t sys_err, std::string err_msg) {
-  CDM_DLOG() << "OpenCdm::ErrorCallback";
+void OpenCdm::ErrorCallback(OpenCdmPlatformSessionId,
+    uint32_t, std::string err_msg) {
```
Comment (calvaris) [inline src/browser/wpe/opencdm/open_cdm.cpp:286]: This could be single line.

File: src/browser/wpe/opencdm/open_cdm.cpp [hunk 3]
```diff
@@ -250,86 +240,82 @@ int OpenCdm::ReleaseMem() {
...
   m_eState = KEY_SESSION_ERROR;
   m_cond_var.notify_all();
+  CDM_LOG_LINE("call over");
 }
 
-void OpenCdm::MessageCallback(OpenCdmPlatformSessionId platform_session_id,
-    std::string& message, std::string destination_url) {
+void OpenCdm::MessageCallback(OpenCdmPlatformSessionId,
+                              std::string& message,
+                              std::string destination_url)
```
Comment (calvaris) [inline src/browser/wpe/opencdm/open_cdm.cpp:297]: This could be a single line.

File: src/com/cdm/rpc/rpc_cdm_platform_handler.cc [hunk 4]
```diff
@@ -103,24 +109,26 @@ RpcCdmPlatformHandler::RpcCdmPlatformHandler(
...
-                      call_params);
-  CDM_DLOG() << "thread created: " << rc;
+  if (!pthread_create(&thread1, NULL, RpcCdmPlatformHandler::DelegateRpcInit, call_params)) {
+      CDM_LOG_LINE("failed to create a thread: %s", strerror(errno));
+      return;
+  }
 
   com_state = INITIALIZED;
-  // TODO(sph): pthread_exit to terminate thread
+  // TODO(sph): pthread_exit to terminate thread // Oh dear.
```
Comment (calvaris) [inline src/com/cdm/rpc/rpc_cdm_platform_handler.cc:131]: lol

**Review by calvaris — APPROVED** [review]

File: src/browser/wpe/opencdm/open_cdm.cpp (hunk 1 above)
Comment (charlie-ht) [reply src/browser/wpe/opencdm/open_cdm.cpp:161]: Yes, yes it does :)

**Pull request merged**